doc/Language/slangs.rakudoc

```diff
@@ -0,0 +1,168 @@
+}
+
+my role LegacyLambda {
+    token lambda { '->' | '<->' | 'λ' }
+}
+
+use Slangify:ver<0.0.4+>:auth<zef:lizmat> RakuLambda, Mu, LegacyLambda;
+=end code
+
+Then to use it, we do:
+
+=begin code
+use Slang::Lambda;
+
+say (1,2,3).map: λ $x { $x+1 }  # (2 3 4)
+
+=end code
+
+The code is self-evident to those familiar with Raku's grammars.  
+This is all well and good, but sometimes there's a large gap between 
+being able to read code, and being able to write it.  How did Elizabeth (the
+author of Slang::Lambda) know to use Slangify?  How did she know what token 
+names to declare?  In her case, it was through extensive study of Raku and
+how it works, but in order to avoid this, we're going to step through some
+of these things now.  
+
+=head2 Slangify
+
+To ease the process of creating Slangs, Elizabeth also created the 
+L<Slangify|https://raku.land/zef:lizmat/Slangify> module.  The Slangify module 
+is a module intended for slang developers.
+
+It abstracts the internals of slang creation and activation so that module 
+developers of slangs have a consistent interface they can work with across 
+current and future versions of the Raku Programming Language.  
+
+To use it you do the following:
+
+=begin code
+use Slangify Foo::Grammar, Foo::Actions;
+=end code
+
+Either the Grammar or the Actions can be replaced with C<Mu> to indicate that
+they should remain unchanged.  
+
+Two additional parameters, for legacy grammars and legacy actions, are also
+accepted.  
+
+While using Slangify is logically the last step in your code, it's worth 
+understanding it somewhat before you begin, so that you can align your
+code with it.  For further details, read the documentation for 
+L<Slangify|https://raku.land/zef:lizmat/Slangify>.  
+
+=head2 The Raku Grammar
+
+This is the point where you need to do the most digging.  The Raku grammar
+is not small, and nor is it simple; its extensive error messages, which 
+are great when programming, hinder the readability.  
+
+The sources for the Raku grammar are:
+
+=item1 L<The new AST grammar|https://github.com/rakudo/rakudo/blob/main/src/Raku/Grammar.nqp>
+
+=item1 L<The old, pre-AST grammar|https://github.com/rakudo/rakudo/blob/main/src/Perl6/Grammar.nqp>
+
+Unfortunately there's no substitute for digging through the grammar and 
+locating the tokens/rules which you wish to override.  
+
+=head2 The Slang Grammar
+
+Now you need to write the slang grammar, much like the one that Elizabeth wrote 
+for Slang::Lambda.  The links above to the grammar documentation should help
+here.  
+
+=head2 The Slang Actions
+
+Many slangs will also need an Actions class backing them.  This is also 
+documented in the grammar documentation linked above.  
+
+=head2 Publishing the module
+
+You'll want to get your module published too.  To do this, follow the 
+instructions under L<Upload your module to zef ecosystem|https://docs.raku.org/language/modules#Upload_your_module_to_zef_ecosystem>.  
```

Comment:
any links to docs.raku.org should be relatively links so a pod renderer for the project has a chance to deal with the links locally.

```
L<...|/language/modules#Upload_your_module_to_zef_ecosystem>
```